### DISCUSSION TITLE
navigation through episodes in manual search

### DISCUSSION THREAD

**Discussion by Anatharias** [pymedusa/Medusa#7133] [Ideas & Feature requests]
Currently, if you want to add a show from the manual selection page, you must go back to the main show page then click the next episode manual icon.

Wouldn't it be easier if a "Previous episode" and "Next episode" buttons would be available in the manual search page to easily go from one episode to the next and do whatever needs to be done in that manual search page?

the two buttons bottom right are only for navigating in the table if it is too long (silly).

**Comment by p0psicles** [comment]
You shouldn't really need to do that for allot of episodes. I use manual search to fine tune the searching. Like check what kind of releases are available and configure quality, required or ignored words for that show.

And then start one forced backlog search. And if that works queue the rest.

**Comment by Anatharias** [comment]
No, indeed I shouldn't. But the fact is that I must quite often because my search terms and favorite terms are not found even though torrents are available... so sometimes, I'm just letting it go and manually add torrents, it is fine, quite frankly, this is alright. But switching from ep 1 to ep 2 etc is really too slow, a quick "next" or "previous" button , which I think wouldn't be too difficult to program, would be a nice addition to the manual search.

Also, you have to take into account that non-english shows / tems are not really Medusa's strong suit

**Comment by 007revad** [comment]
When I need to do that I just right-click on the manual search button and select "open in new tab" from the context menu. Then repeat for all the other episodes I want to search for. Each tab automatically waits its turn to search as Medusa only allows 1 tab/window to search at a time.
